src/locales/fr.json

```diff
@@ -75,13 +75,14 @@
     "primary_biometry": "Déverrouiller avec mes empreintes biométriques",
     "primary_faq": "Manuel d'utilisation",
     "primary_lock_screen": "Écran de verrouillage",
+    "primary_lock_switch": "Verrouiller en passant l’app en arrière plan",
```

Comment:
Wondering about this.

Our figma spec says `Vérrouiller en quittant l'app`
Our trello card says `Vérrouiller en quittant l'app ou en la passant en arrière plan`
This PR says `Verrouiller en passant l'app en arrière plan`